bobtemplates/hooks.py

```diff
@@ -49,3 +51,22 @@ def get_plone_classifier_version(configurator, question, answer):
     configurator.variables['package.classifier_version'] = '.'.join(an_split[:2])
 
     return answer
+
+
+def get_deploy_scripts(configurator):
+    deploy_scripts = {
+        'pull': requests.get('https://raw.githubusercontent.com/4teamwork/plone-git-deployment/master/deploy/pull'),
+        'after_push': requests.get('https://raw.githubusercontent.com/4teamwork/plone-git-deployment/master/deploy/after_push'),
+        'update_plone': requests.get('https://raw.githubusercontent.com/4teamwork/plone-git-deployment/master/deploy/update_plone')
+    }
+
+    path = os.path.join(os.getcwd(),
+                        'generated',
+                        configurator.variables['package.fullname'],
+                        'deploy')
+
+    os.makedirs(path)
+
+    for k, v in deploy_scripts.iteritems():
+        file = open(os.path.join(path, k), 'w')
```

Comment:
1. please do not use `file` as variable names, as this is reserved because it is a [built-in](https://docs.python.org/2/library/functions.html).
2. Please use `open` as context manager in order to make sure that the file handle will be closed.